app/main/views/users.py

```diff
@@ -30,7 +44,7 @@ def auth_user():
         db.session.add(user)
         db.session.commit()
 
-        return jsonify(users=user.serialize()), 200
+        return single_result_response("users", user), 200
```

Comment:
Another one that should be `RESOURCE_NAME`.

Reply:
Nice